patches/content-browser-worker_host-shared_worker_host.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/content/browser/worker_host/shared_worker_host.h b/content/browser/worker_host/shared_worker_host.h
+index 6e1930d3ed50692ce5a0daea77648aea0b52e0b0..da4a49057e4dd3fb8a17cc47d4e50d3669fb48a7 100644
+--- a/content/browser/worker_host/shared_worker_host.h
++++ b/content/browser/worker_host/shared_worker_host.h
+@@ -197,6 +197,7 @@ class CONTENT_EXPORT SharedWorkerHost : public blink::mojom::SharedWorkerHost,
+   // Creates a network factory params for subresource requests from this worker.
+   network::mojom::URLLoaderFactoryParamsPtr
+   CreateNetworkFactoryParamsForSubresources();
++  BRAVE_SHARED_WORKER_HOST_H
```

Comment:
please revisit these patches. I think some of them can be replaced with redefining existing functions or members (`#define FunctionName ...`).

macro insertion should go right next to `private:`/`protected:` lines, i.e. no empty lines between:
```
  CreateNetworkFactoryParamsForSubresources();

  BRAVE_SHARED_WORKER_HOST_H
 private:
```

Reply:
Removed one header patch altogether and fixed the spacing on the others. Most of the header file classes only have overrides which (AFAIK) we can't use for preprocessor tricks because it makes the ancestor classes fail to compile.

Reply:
It's a Fintech company
 
 Register with the link and visit "Neufund"  https://neufund.co/r/?a=jamesbethel66@gmail.com

It's actually a cryptocurrency investment company , they also operate on real estate, oil asset and mortgages as a back up financial aid